app/src/main/java/com/angdroid/refrigerator_manament/domain/usecase/GetDefaultFoodDetailUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package com.angdroid.refrigerator_manament.domain.usecase
+
+interface GetDefaultFoodDetailUseCase { //데이터베이스상 대표 음식 (ex: 사과,당근,생닭) 이름과 이미지를 가져오는 함수
```

Comment:
하지만 더 좋은 방법으로는 메소드 주석 방식을 사용하는 것입니다!
```suggestion
/**
 * 음식 카테고리(닭,계란,당근,오이)를 가져오는 함수(사용자의 음식리스트에 쓰일 예정)
 */
interface GetDefaultFoodDetailUseCase { 
```
다음과 같이 해둔다면 해당 함수에 커서를 올려두거나 Window기준 Control+Q을 누른다면 문서 조회로 더 좋은 주석이 될 수 있습니다! @Param 등 추가할 수 있는게 많으니 한 번 알아보세요!!
<img width="517" alt="스크린샷 2022-10-11 오전 1 30 14" src="https://user-images.githubusercontent.com/15981307/194913076-40095359-1be7-40a4-a072-8cbdae16157a.png">